.github/workflows/ci-sphinx.yml

```diff
@@ -15,7 +15,12 @@ jobs:
     name: Sphinx
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - uses: ammaraskar/sphinx-action@master
-      with:
-        docs-folder: "docs/"
+
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Build docs
+      run: |
+        sudo apt-get install -y python3-sphinx
+        pip3 install sphinx-rtd-theme
```

Comment:
We should install there everything from `docs/requirements.txt` in order to be compatible with how RTD builds the docs (for potential future changes)